tests/baselines/reference/tscWatch/resolveJsonModule/incremental-always-prefers-declaration-file-over-document.js

```diff
@@ -103,6 +108,11 @@ var x = data_json_1.default;
         "./main.ts"
       ]
     ],
+    "referencedMap": {
+      "./main.ts": [
+        "./data.d.json.ts"
+      ]
+    },
```

Comment:
This feels like an improvement - even though there is a resolution error here, it still depends on this file so it should be reflected here. IIRC, there are other instances of something like this - when despite a resolution errors requested modules are listed here.